### PR TITLE
VSTS-438 - User Notifications

### PR DESCRIPTION
**Pull request by jessedoyle**
:art:

#### Details

_TODO_

#### Screenshot

https://user-images.githubusercontent.com/6474230/27752159-ce146a02-5d9c-11e7-9878-9328c0ed474a.png

https://user-images.githubusercontent.com/6474230/27752165-d4b371e6-5d9c-11e7-83aa-205bbbe1e4fe.png

### CODE REVIEW

File: app/assets/stylesheets/layout_components/notices.scss [hunk 1]
```diff
@@ -0,0 +1,113 @@
+.notice--flash{
+  padding: $base-padding/2 $base-padding;
+  background-color: $brand-blue-insurance;
+  color: $white;
+  -webkit-animation: noticeFade 8s;
+  -moz-animation: noticeFade 8s;
+  -o-animation: noticeFade 8s;
+   animation: noticeFade 8s;
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/layout_components/notices.scss]: This looks too indented.

**Comment by MathieuGilbert** [comment]
💩

**Comment by jessedoyle** [comment]
:wave:

**Comment by jessedoyle** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:art: :art: :art:

**Pull request merged**